pkg/uroot/uroot.go

```diff
@@ -138,6 +134,32 @@ func ResolvePackagePaths(env golang.Environ, pkgs []string) ([]string, error) {
 				importPaths = append(importPaths, p.ImportPath)
 			}
 		}
+		return importPaths, nil
+	}
+
+	// No file import paths found.
+	// TODO(subhavr): Now that we look in all src dirs, I don't think this can succeed:
```

Comment:
You might be right. I'd still leave it here, as I'm not sure exactly what all Package() does under the hood.

Reply:
Done.